### PR TITLE
proxi command for removing a chain

### PR DESCRIPTION
**Pull request by HM4704**

### CODE REVIEW

File: api/client/client.go [hunk 1]
```diff
@@ -574,6 +574,102 @@ func (c *APIClient) MakeChainOrigin(par TransferFromED25519WalletParams) (*trans
...
+
+func (c *APIClient) RemoveChainOrigin(par RemoveChainOriginParams) (*transaction.TxContext, error) {
+	if par.TagAlongFee > 0 && par.TagAlongSeqID == nil {
+		return nil, fmt.Errorf("tag-along sequencer not specified")
+	}
+
+	walletAccount := ledger.AddressED25519FromPrivateKey(par.WalletPrivateKey)
+
+	// find chain output by chain ID
+	outs, err := c.GetAccountOutputs(walletAccount, func(_ *ledger.OutputID, o *ledger.Output) bool {
```
Comment (lunfardo314) [inline api/client/client.go:593]: with GetAccountOutputs is one way of doing this but is suboptimal. It essentially is filtering all outputs in the address account and finding the right one.
The optimal way of finding chain output is with  GetChainOutputFromHeaviestState. It retrieves raw output data for the given chain ID. It always is unique (if exists at all).
Then there will be no need for the subsequent logic of parsing, finding etc

File: proxi/node_cmd/rmchain.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	"github.com/lunfardo314/proxima/ledger/transaction"
+	"github.com/lunfardo314/proxima/proxi/glb"
+	"github.com/lunfardo314/proxima/util"
+	"github.com/spf13/cobra"
+)
+
+func initRemoveChainCmd() *cobra.Command {
+	removeChainCmd := &cobra.Command{
+		Use:   "rmchain <chain id>",
+		Short: `removes a chain origin (not a sequencer)`,
```
Comment (lunfardo314) [inline proxi/node_cmd/rmchain.go:18]: I see where 'remove' is coming from :) Still, the semantics of the operation is different from the one of files. We are not just removing output, we are discontinuing the chain

Comment (lunfardo314) [reply proxi/node_cmd/rmchain.go:18]: It discontinues any chain, also sequencer chain. It is up to the controller if he wants to create a mess by destroying the chain of its own sequencer. But I wouldn't worry about it now

File: api/client/client.go (hunk 1 above)
Comment (HM4704) [reply api/client/client.go]: Done.

File: proxi/node_cmd/rmchain.go (hunk 2 above)
Comment (HM4704) [reply proxi/node_cmd/rmchain.go]: Done.

**Review by lunfardo314 — APPROVED** [review]

**Pull request merged**